test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java

```diff
@@ -58,7 +58,10 @@ public static void main(String... args) {
 	private HapiApiSpec accountsGetPayerRecordsIfSoConfigured() {
 		return defaultHapiSpec("AccountsGetPayerRecordsIfSoConfigured")
 				.given(
-						cryptoCreate("payer")
+						cryptoCreate("payer"),
+						fileUpdate(APP_PROPERTIES)
+								.payingWith(ADDRESS_BOOK_CONTROL)
+								.overridingProps(Map.of("ledger.keepRecordsInState", "false"))
```

Comment:
This step in done in then().

Reply:
I will check the other related PR to make change if necessary

Reply:
In JRS, "ledger.keepRecordsInState" was set to true before launch the client.
This test case accountsGetPayerRecordsIfSoConfigured  can only pass if before 
ledger.keepRecordsInState is set to false.

Reply:
sorry for the late reply Jeffrey, as by default now that ledger.keepRecordsInState=true, when this is set to false accounts in records must be zero. That is why I set this value false in when(). You are just repeating the transaction in given. So it just will be repeated operation in when()

Reply:
Got it, I will revert the change later

Reply:
But, was it still failing with my changes in ?